src/flashFormatter/H7FlashFormatter.cpp

```diff
@@ -0,0 +1,170 @@
+/*
+  Copyright (c) 2025 Arduino SA
+
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+#if defined(ARDUINO_PORTENTA_H7_M7) || defined(ARDUINO_NICLA_VISION) \
+  || defined(ARDUINO_OPTA) || defined(ARDUINO_GIGA)
+#include "H7FlashFormatter.h"
+#include "wiced_resource.h"
+#include "certificates.h"
+
+//QSPIFBlockDevice _root(QSPI_SO0, QSPI_SO1, QSPI_SO2, QSPI_SO3,  QSPI_SCK, QSPI_CS, QSPIF_POLARITY_MODE_1, 40000000);
+
+MBEDH7FlashFormatter::MBEDH7FlashFormatter():
+_root(QSPI_SO0, QSPI_SO1, QSPI_SO2, QSPI_SO3,  QSPI_SCK, QSPI_CS, QSPIF_POLARITY_MODE_1, 40000000),
```

Comment:
why not use also here get_default_instance ?

Reply:
I'm not sure if it's the same, I used the same configuration of the  QSPI format sketch

Reply:
yes it works the same for stm32H7, i would use the same approach of c33 with `get_default_instance();`